book/scheduling.md

```diff
@@ -383,7 +384,7 @@ do security checks:
 
 ``` {.python}
 class JSContext:
-    def XMLHttpRequest_send(self, method, url, body, is_async, handle):
+    def XMLHttpRequest_send(self, method, url, body, isasync, handle):
```

Comment:
Why this change?

Reply:
lol, these code blocks were one character too wide, and I didn't want to spill the arguments over two lines.